src/Chinchilla/Publisher.cs

```diff
@@ -128,6 +128,7 @@ public override void Dispose()
 
             ModelReference.Dispose();
             disposed = true;
+            base.Dispose();
```

Comment:
I would set the `disposed = true` flag after the base dispose runs, just because it's not really disposed until then...